### ISSUE TITLE
next Arcadia Sprint at CERN, September 16-20

### ISSUE THREAD

**Issue by myrmoteras** [plazi/arcadia-project#28]
Dear @tcatapano @punkish @gsautter @teodorgeorgiev @guidoti @lnielsen 

We are running our next Sprint as discussed between September 16-20, 2019 at CERN. Please confirm your participating asap.

The days before there is open day at CERN. For those interested, please visit https://home.cern/news/news/cern/cern-open-days-explore-future-us and let us know.

We need to reserve rooms. I am not sure whether we can still get rooms at CERN hostel, or whether we should look into the IBIS Budget. Let me know what you prefer. 
I personally liked the hikes which gave a good chance for exchanges and lengthy discussions. There are still rooms at the IBIS, so, please reserve them asap for you.

Is there anybody you would like to invite, similar to Tim?

The issue came up that we need to widen the audience and thus consider to provide a training course or similar to use these new tools we provide. Although this might be on a too short note for the Fall Sprint, I would like to get your idea whether we should consider this for the next Sprint in Spring 2020.

thanks for your input

**Comment by lnielsen** [comment]
You mean September 9th - 13th right?

**Comment by myrmoteras** [comment]
No, I have in my note that we run the workshop after the CERN open day.

Do you want to run it before. We can still change it.

**Comment by lnielsen** [comment]
I'm quite sure on the last day we agreed to have it the week before :-) At least, that's the date that first me the best and the days I've already booked the sprint room for. So if possible I would highly prefer having it September 9-13.

**Comment by lnielsen** [comment]
See #15

**Comment by myrmoteras** [comment]
no problem - let's move it to 9-13.

**Comment by myrmoteras** [comment]
So, please all note: 

Next sprint September 9-13!

**Comment by myrmoteras** [comment]
Please reply by Friday March 1 whether you will participate at the Sprint so I can book the rooms
@teodorgeorgiev @punkish @gsautter @guidoti @tcatapano.
The booking will be from Sunday Sept 8 to Saturday 14, unless you want to leave on Friday evening or spend another day and leave on Sunday.
Sept 14/15 is open day at CERN.

**Comment by guidoti** [comment]
We talked before and today, I'm waiting Plazi's decision on my participation, that's why I didn't say anything before. Let me know!

**Comment by gsautter** [comment]
I'd like to stay there Sept 8-15, would like to see the CERN open house without having to rush off.

**Comment by tcatapano** [comment]
+1 count me in. I'd also like to stay to the 15th.

**Comment by gsautter** [comment]
Also, please add +1 for Johannes, maybe Sept 10 or 11 to 15, as I'm sure we'll want to see the CERN open house with us, apart from meeting the crew.

**Comment by teodorgeorgiev** [comment]
+1, I'd like to stay Sept 9-13

**Comment by myrmoteras** [comment]
The hotel is booked according the list below. It is a flexible tarif and with member tarif. It does not inlude breakfast, since most of us only drink a coffee or tea to start.

Donat 8.9. - 14.9.
Terry 8.9. 15.9.
Guido 8.9. 15.9.
Johannes 10.9. 15.9.
Teodor 8.-13.
Puneet	8.-15.
Marcus	8.-15.

**Comment by mguidoti** [comment]
Just a reminder to whom might be interested: https://opendays-registration.cern.ch/

**Comment by myrmoteras** [comment]
@punkish @mguidoti @tcatapano 

here are the train schedules and tickets:
* Sunday Sept 8 12:59pm Paris Gare de Lyon - Lyon - Geneva (17:08) (Donat, Puneet, Marcus) [Ticket](https://github.com/plazi/arcadia-project/files/3546715/TGV_Paris_Geneve_201909081259_RUXFWD_Donat_Marcus_Puneet.pdf)

* Sunday Sept 15 13:41 Geneva - Paris Gare de Lyon 16:55  [Ticket](https://github.com/plazi/arcadia-project/files/3546696/TGV_Geneve_Paris_201909151341_RVUHIK_Terry_Marcus.pdf)

**Comment by myrmoteras** [comment]
social dinner: Tuesday night, organized by @lnielsen or @slint
